.github/workflows/release.yml

```
@@ -10,9 +10,10 @@ concurrency:
   cancel-in-progress: false  # Never cancel releases
 
 permissions:
-  contents: write  # Required for creating releases
-  packages: write  # Required if publishing to GitHub Packages
-  id-token: write  # Required for signing
+  contents: write   # Required for creating releases and pushing tags
+  packages: write   # Required if publishing to GitHub Packages
+  id-token: write   # Required for signing
+  workflows: write  # Required for pushing tags that reference commits with workflow changes
```

actionlint (1.7.10)

[error] unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)